### DISCUSSION TITLE
WYSISYG Table editor has to way to resize table.

### DISCUSSION THREAD

**Discussion by CoffeeCodeAndCreatine** [requarks/wiki#3584] [Error / Bug Report]
**Describe the bug**
Table column width cannot be resized and table cannot exceed the width of a page. 
Table columns should be able to expand, and table should be able to scroll horizontally.

**To Reproduce**
1. Create page with WYSIWYG editor
2. Create a table with 17 columns and type "this is some really long data" in each
3. Save the page and view
4. The table will extend past the edge of the page with no way to see the contents.

**Expected behavior**
I expect to be able resize tables so that I can see the data easily. 
I expect to be able to scoll in a table both vertically and horizontally.

**Comment by Smankusors** [comment]
could be related with #1314.

An alternate way to do this, using raw HTML instead.
